app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :post, class_name: 'Post'
+  belongs_to :user, class_name: 'User'
+
+  def update_likes_counter
+    post.update(likes_count: post.likes.count)
+  end
+end
```

Comment:
[OPTIONAL] It is a good practice to run methods like this with rails callbacks like `:after_save` and make the method private.